sorc/ncep_post.fd/CALUPDHEL.f

```diff
@@ -1,39 +1,18 @@
 !> @file
-!
```

Comment:
@kayeekayee It seems to me your branch revert the changes in PR #458. Can you modify?